Add docs for skip control plane ip check annotation (#6528)
aws/eks-anywhere: docs/content/en/docs/getting-started/optional/annotations.md

---
title: "Annotations configuration"
linkTitle: "Annotations"
weight: 45
aliases:
    /docs/reference/clusterspec/optional/annotations/
description: >

>  EKS Anywhere cluster yaml annotations to skip validations

Reviewer: ```suggestion
 EKS Anywhere cluster annotations to skip validations
```

---

> EKS Anywhere runs preflight validations while creating the EKSA cluster amongst one of which is checking if the control plane IP is a valid IP that can be used or not. This validation is only performed while creating a cluster. It is also only applicable to all production grade providers.

Reviewer: ```suggestion
EKS Anywhere runs validations while performing cluster operations. To skip some of these validations, you can configure the following annotations: 
```

> We introduce an annotation, that is particularly more helpful while creating workload clusters using tools like kubectl, to skip the validation from running.

Reviewer: ```suggestion
```
